test/test_utils/eks.py

```diff
@@ -215,10 +228,29 @@ def delete_cfn_stack_and_wait(stack_name):
         LOGGER.error(f"Error: Cannot delete stack: {stack_name}. Full Exception:\n{e}")
         describe_cfn_stack_events(stack_name)
 
+def delete_oidc_provider(eks_cluster_name):
+    """Function to delete the oidc provider created by kubeflow
+    Args:
+        eks_cluster_name: str
+    """
+    eks_client = init_eks_client()
+    iam_client = init_iam_client()
```

Comment:
nit: Why have separate functions for these? It seems kind of redundant, because it's still a one-line function call.

not nit: Please create the EKS client with a region (afaik, eks clients are regional, right?), with the region set to test_utils.REGION by default, and any other region if an override is required. We want to make the code region-agnostic as far as possible in case we'd like to run tests in multiple regions in the future.

Reply:
+1

Reply:
Modified accordingly